tests/core/editable/keystrokes/delbackspacequirks/manual/nbspbackspacing.html

```diff
@@ -0,0 +1,11 @@
+<textarea id="editor" cols="30" rows="10">
+	<p>Hello World!</p>
+</textarea>
+
+<script>
+	if ( !CKEDITOR.env.webkit ) {
```

Comment:
I wasn't able to reproduce this issue on Safari, so I guess it's better to use `CKEDITOR.env.chrome` here? (it will still accept the new Edge and it should as the issue is also present there).

Reply:
Oh, I see it is reproducible in Safari and I've omitted it because in Safari `14.1` devtools the `&nbsp;` is displayed simply as ` ` 👎🏻 so it can only be verified through the source mode. Therefore this comment is invalid, please leave the flag as is.